lib/signs_ui_web/templates/signs/index.html.eex

```diff
@@ -1,4 +1,3 @@
-<h3>Countdown Signs</h3>
```

Comment:
I think we should *keep* this one, and get rid of the "MBTA Realtime Signs" header above, rather than vice versa. Since the param is a property of _this_ controller, it seems like the route should be a property of this template. Presumably, if the app grew to have other controllers or concerns, we wouldn't want the route header up in the top level layout template.

Reply:
I kind of agree and thats how i had it originally, but the mockup has the main header being the one thats replaced so i tried to match that.

Reply:
if you look at the previous commit, that is how it was - would you rather we went with that approach?

Reply:
I see... Well, if we get rid of the other one entirely, can we make this H3 and H1 and therefore be true to the mockup? I don't think there's usually headers on layout templates, so it'd be fine to put the H1 in the view template.

Reply:
ok, thats fine with me